src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -256,8 +256,11 @@ public void GetTempFileNameTempInvalidTemp()
                     File.Delete(badTemp);
                     Assert.Throws<DirectoryNotFoundException>(() => Path.GetTempFileName()); // non existent
 
-                    Environment.SetEnvironmentVariable(tempEnvVar, "|||");
-                    Assert.Throws<IOException>(() => Path.GetTempFileName()); // invalid path
+                    if (OperatingSystem.IsWindows())
```

Comment:
On Unix of course this is a valid path. I don't see an easy way to get GetTempFileName to throw IOException on Unix. Which is fine, because there's no loop in the implementation on Unix running when there's an IOException.

Reply:
Should we make an `else` case and verify the Unix behavior?

Reply:
It's a valid path in Unix, just doesn't exist, so it's the same as the case above. We have other tests for what paths are valid on Unix

Reply:
Ah, good point. It might make sense to stick that in a comment, but I wouldn't reset CI for it.